js/google-map.js

```diff
@@ -1,9 +1,75 @@
-(function ($, Drupal) {
+(function ($, Drupal, navigator) {
 
   "use strict";
 
   if (!$('.geolocation-location').length && $('.geolocation-location').length !== -1 && $('.view-id-search_solr_.view-display-id-attachment_1').length) {
     $('.view-id-search_solr_.view-display-id-attachment_1').hide();
   }
 
-})(jQuery, Drupal);
+  getclientlocation();
+
+  function getclientlocation () {
+    // If the browser supports W3C Geolocation API.
+    if (navigator.geolocation) {
+
+      var $latElement = $('input[name="center_lat"]');
+      var $lngElement = $('input[name="center_long"]');
+      var $currentlocation = $('#edit-current-location');
+
+      // Get the geolocation from the browser.
+      navigator.geolocation.getCurrentPosition(
+
+        // Success handler for getCurrentPosition()
+        function (position) {
+          var lat = position.coords.latitude;
+          var lng = position.coords.longitude;
+          var accuracy = position.coords.accuracy / 1000;
```

Comment:
Is this going to be used for anything?

Reply:
oops thanks for noticing it, will remove.